content/trial_getting_started/set_trial_manager/other-deployments.md

```diff
@@ -19,7 +19,8 @@ Read about the differences between our {{< param product_name >}} versions, and
 
 | Version | Release date | Release Notes | Manager RPM | Manager Container | Manager Image | CLI RPM | CLI DEB | CLI EXE | End of life |
 |:-------:|:-----:|:-------------:|:-----------:|:-----------------:|:-------------:|:-------:|:-------:|:-------:|:-------:|
-| **6.2.0**  | September 23ed, 2021 | [6.2.0 RN](/pdf/620RN.pdf) | [RPM](https://repository.cloudifysource.org/cloudify/6.2.0/ga-release/cloudify-manager-install-6.2.0-ga.el7.x86_64.rpm)	|	[Docker (All-in-one)](https://repository.cloudifysource.org/cloudify/6.2.0/ga-release/cloudify-manager-aio-docker-6.2.0-ga.tar)	|	[QCOW](https://repository.cloudifysource.org/cloudify/6.2.0/ga-release/cloudify-manager-premium-6.2.0.qcow2) | [RPM](https://repository.cloudifysource.org/cloudify/6.2.0/ga-release/cloudify-cli-6.2.0-ga.el7.x86_64.rpm)	|	[DEB](https://repository.cloudifysource.org/cloudify/6.2.0/ga-release/cloudify-cli_6.2.0-ga_amd64.deb)	|	[EXE](https://repository.cloudifysource.org/cloudify/6.2.0/ga-release/cloudify-windows-cli_6.2.0-ga.exe) | Oct 23, 2023 |
+| **6.3.0**  | February 10th, 2022 | [6.3.0 RN](/pdf/630RN.pdf) | [RPM](https://repository.cloudifysource.org/cloudify/6.3.0/ga-release/cloudify-manager-install-6.3.0-ga.el7.x86_64.rpm)	|	[Docker (All-in-one)](https://repository.cloudifysource.org/cloudify/6.3.0/ga-release/cloudify-manager-aio-docker-6.3.0-ga.tar)	|	[QCOW](https://repository.cloudifysource.org/cloudify/6.3.0/ga-release/cloudify-manager-premium-6.3.0.qcow2) | [RPM](https://repository.cloudifysource.org/cloudify/6.3.0/ga-release/cloudify-cli-6.3.0-ga.el7.x86_64.rpm)	|	[DEB](https://repository.cloudifysource.org/cloudify/6.3.0/ga-release/cloudify-cli_6.3.0-ga_amd64.deb)	|	[EXE](https://repository.cloudifysource.org/cloudify/6.3.0/ga-release/cloudify-windows-cli_6.3.0-ga.exe) | Oct 23, 2023 |
```

Comment:
`https://repository.cloudifysource.org/cloudify/6.3.0/ga-release/cloudify-manager-aio-docker-6.3.0-ga.tar`
should be instead
`https://repository.cloudifysource.org/cloudify/6.3.0/ga-release/cloudify-manager-aio-docker-6.3.0-ga-x86_64.tar` (for intel/amd x64 architecture), and
`https://repository.cloudifysource.org/cloudify/6.3.0/ga-release/cloudify-manager-aio-docker-6.3.0-ga-aarch64.tar` (for apple M1 architecture)